English/QUEST_JOBSTEP.tsv

```diff
@@ -3128,13 +3128,13 @@ QUEST_JOBSTEP_20150803_003127	Search for the traces of Lydia Schaffen
 QUEST_JOBSTEP_20150803_003128	The Fletcher Master wants you to look for the bow that the first Fletcher Master made for Lydia Schaffen. Find the bow from 31_1 of Canyon Area.
 QUEST_JOBSTEP_20150803_003129	Give them to the Fletcher Master
 QUEST_JOBSTEP_20150803_003130	You've found the bow of the first Fletcher Master. Return to the current Fletcher Master.
-QUEST_JOBSTEP_20150811_003131	Selamat datang. {nl}Kita kehabisan banyak hal, jadi kita harus memanfaatkan semua yang kita punya.
-QUEST_JOBSTEP_20150811_003132	Mothstem? Aku pernah melihatnya di Fedimian Suburbs.{nl}Pardoner Master juga sedang mencarinya. Apa kamu belum menemukannya?
-QUEST_JOBSTEP_20150811_003133	Cepat kembali ke Pardoner Master. Ia sedang mencari Jar of Fire itu sejak lama.
-QUEST_JOBSTEP_20150811_003134	Maksudmu Jar of Fire? Aku tahu. {nl}Tapi aku tidak akan memberitahumu secara cuma - cuma. Mungkin hardened oil dari Blue Puragi di Saknies Plains dapat membuatku berbicara...
-QUEST_JOBSTEP_20150811_003135	Bila kamu tidak membutuhkannya, tidak ada pilihan lain. {nl}Aku adalah seorang merchant, bukan sukarelawan.
-QUEST_JOBSTEP_20150811_003136	Oh, terima kasih banyak. {nl}Kudengar bahwa sesosok monster bernama Mothstem memiliki Jar of Fire dan aku takkan pernah melepaskannya.
+QUEST_JOBSTEP_20150811_003131	Welcome. {nl}We're running a lot of things, so we must take advantage of all we got.
```

Comment:
> _Kita kehabisan banyak hal_
_We're running **out** a lot of things_ (kurang out)

> _harus memanfaatkan semua yang kita punya_
_must take advantage of all we **got**_ (bisa diubah jadi **have** kalo mau agak formal)